x-pack/plugins/observability_solution/observability_onboarding/public/assets/integrations.conf

```diff
@@ -0,0 +1,88 @@
+[system]
+title=System Logs And Metrics
+# patterns=
+#     /var/log/system.log*
+
+[nginx]
+title=Nginx Logs
+patterns=
+    /var/log/nginx/access.log*
+    /var/log/nginx/error.log*
+
+[apache]
+title=Apache Logs
+patterns=
+    /var/log/apache2/access.log*
+    /var/log/apache2/other_vhosts_access.log*
+    /var/log/apache2/error.log*
+    /var/log/httpd/access_log*
+    /var/log/httpd/error_log*
+
+[docker]
+title=Docker Container Logs
+patterns=
+    /var/lib/docker/containers/*/*-json.log
+    /var/run/docker.sock
+
+[mysql]
+title=Mysql Logs
```

Comment:
Same as the badges on the auto-detection page. 

Can you use the correct casing for the integrations?